apis/dscinitialization/v1/dscinitialization_types.go

```diff
@@ -33,21 +34,34 @@ type DSCInitializationSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,order=2
 	// +optional
 	Monitoring Monitoring `json:"monitoring,omitempty"`
-	// Internal development useful field
+	// Internal development useful field to test customizations. This is not recommended
+	// to be used in production environment.
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,order=3
 	// +optional
-	ManifestsUri string `json:"manifestsUri,omitempty"`
+	DevFlags DevFlags `json:"devFlags,omitempty"`
```

Comment:
just to clarify: if it is called `devFlags` or `devflags` (which is set in the oprator working session doc)

Reply:
Yes its uniform with the other fields that we have